io/net/port_bounce.py.data/README.txt

```diff
@@ -0,0 +1,13 @@
+Port bounce Testcase:
+
+this testcase mainly depends on the setup of fc or fcoe switch to
+connect different adapters to different ports in respective switches.
+
```

Comment:
Test can be improved to also do a storage side port off on ?

[Server] <  p1- [switch] - p4 >   [storage]